biz.aQute.bnd/bnd.bnd

```diff
@@ -44,7 +44,10 @@ Bundle-Description: This command line utility is the Swiss army knife of OSGi. I
 	biz.aQute.remote.api;version=latest,\
 	org.yaml.snakeyaml;version=1.15
 
--testpath: ${junit}, biz.aQute.bnd.embedded-repo; version=snapshot
+-testpath: \
+	${junit},\
+	biz.aQute.bnd.embedded-repo; version=snapshot,\
+	org.apache.felix.framework;version=latest;packages=*
```

Comment:
Now that you have removed the direct reference to Felix code (by using ServiceLoader), you should remove `packages=*` from the `-testpath` entry.